### PR TITLE
Filter toast loader

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## FIlter Toast & Loader
- Added explore page with filters
- Added toast

### CODE REVIEW

**Review by MonikaJhingan — APPROVED** [review]
The code is clean and use of various libraries has been done well. Looks good to be merged.

File: src/Components/Card/Card.jsx [hunk 1]
```diff
@@ -71,12 +71,12 @@ const removeFromBookmark = () => {
 
   return (
     <div className=" flex items-center justify-center max-w-[39rem] hover:cursor-pointer">
-      <div className="justify-center p-0.5 max-w-xl border-zinc-400 rounded-[8px]  border-2 dark:text-[#b1b1b1] ">
+      <div className="justify-center p-[0.6rem] max-w-xl border-zinc-400 rounded-[8px]  border-2 dark:text-[#b1b1b1] ">
         <div className=" relative flex items-center justify-between rounded-lg p-2 ">
           <div className="  w-12 flex items-center gap-2 p-1">
-            <img className="rounded-full" src={logo} alt="logo" />
+            <img className=" w-10 h-10 object-contain rounded-full" src={postData?.img} alt="logo" />
```
Comment (Logan1x) [inline src/Components/Card/Card.jsx]: same alt as the above line. May confuse screen readers

File: src/Pages/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -17,13 +17,38 @@ function HomePage() {
   const dispatch = useDispatch();
   const userData = useSelector(state => state.user)
   const {posts} = usePosts();
+  const postData = useSelector(store => store.post);
+  const {allUsers} = useAuth();
   // const posts = useSelector(state=> state.post);
```
Comment (Logan1x) [inline src/Pages/HomePage/HomePage.jsx:22]: Can remove this unused comment.

**Review by Logan1x — APPROVED** [review]
Above mentioned tiny things can be taken care of in the next pull. Everything else looks good.

**Pull request merged**